activity-tracking/server.js

```diff
@@ -6,8 +6,10 @@ require('dotenv').config();
 
 const app = express();
 const port = process.env.PORT || 5300;
-const uri = process.env.MONGODB_URI;
-const mongoUri = config.mongoUri;
+const baseUri = process.env.MONGO_URI || config.mongoUri;
+const database = process.env.MONGO_DB || config.mongoDb;
+const mongoUri = `${baseUri}/${database}?authsource=admin`;
```

Comment:
The activity tracking app would not connect to MongoDB without the authentication source added in. If there are better ways of achieving this let me know.

sources: 
[stackoverflow post](https://stackoverflow.com/questions/75317151/mongoerror-authentication-failed-with-docker-compose)
[change on main repo while the original developer tested changes](https://github.com/CodeFirstGirls/MLA-pilot/commit/5413f3c656685754ef8360c815520c2f827928db#diff-864bc9a3e95275663e7aee1e56a7cd01f1317703dc0ff7a7add96770f6f59d86R10)